docs/wikidata.md

```diff
@@ -0,0 +1,92 @@
+# Wikidata Full Quickstart
+
+The following instructions take you through loading the full Wikidata Knowledge
+Base in QLever. Since Wikidata is very large at over 7 billion triples, **we
+recommend to run all of the following commands in a `tmux` or `screen` session**.
+Nevertheless we will note below when a command will take long to execute.
```

Comment:
`` -> `Commands that take a long time will be marked as such.`
The current sentence implies that the reader will ignore the advice we gave him, which could be considered insulting.